stoservice/src/test/java/com/tutrit/stoservice/settings/ConfigurationsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.tutrit.stoservice.settings;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ConfigurationsTest {
+    Configurations configurations;
+
+    @BeforeEach
+    void setUp() {
+        configurations = new Configurations("src/test/resources/configFalse.properties");
+    }
+
+    @Test
+    void setPathToProperties() {
+        configurations.setPathToProperties("src/test/resources/config.properties");
+        configurations.getProperties();
+        assertTrue(configurations.isLoadDataDemo());
+        configurations.setPathToProperties("src/test/resources/configFalse.properties");
```

Comment:
I would add deleting the "configFalse.properties" file after the test is completed